src/frontend/src/tables/sales/SalesOrderLineItemTable.tsx

```diff
@@ -163,13 +187,15 @@ export default function SalesOrderLineItemTable({
       {
         accessor: 'allocated',
         sortable: true,
-        render: (record: any) => (
-          <ProgressBar
-            progressLabel={true}
-            value={record.allocated}
-            maximum={record.quantity}
-          />
-        )
+        render: (record: any) => {
+          return (
+            <ProgressBar
+              progressLabel={true}
+              value={record.allocated}
+              maximum={record.quantity}
+            />
+          );
+        }
```

Comment:
Question out of curiosity - is there a functional difference between these 2 implementations? I can not see one - eager to improve my js knowledge

Reply:
No, these are functionally equivalent, one returns a component, and the other a function which generates a component. I wonder if one is more "efficient" than the other? In any case, I think that this might have been a change I started to make and then nothing really came of it. I'll revert this diff, there's no need for it.

Reply:
thank you for the explanation - I thought I missed something